framework/src/Volo.Abp.AspNetCore.Mvc/Volo/Abp/AspNetCore/Mvc/Authentication/ChallengeAccountController.cs

```diff
@@ -42,7 +42,12 @@ public async Task<ActionResult> Logout(string returnUrl = "", string returnUrlHa
         {
             await HttpContext.SignOutAsync().ConfigureAwait(false);
 
-            return RedirectSafely(returnUrl, returnUrlHash);
+            if (HttpContext.User.Identity.AuthenticationType == "Identity.Application")
```

Comment:
```
"Identity.Application"
```

hi @gterdem 
This seems to be Identity's cookie scheme. Do we assume that developers are using the identity module?

Reply:
hi @maliming,
Yes. Since it's redirecting to local domain. But I may be missing any other scenarios where developer is not using identity module.